liquibase-extension-testing/src/main/java/liquibase/extension/testing/testsystem/core/FirebirdTestSystem.java

```diff
@@ -15,9 +17,16 @@ public FirebirdTestSystem(Definition definition) {
         super(definition);
     }
 
+    @SuppressWarnings("java:S2095")
```

Comment:
@MalloD12, hello! Please add a comment as to why this Sonar warning is being suppressed. Thank you!

Reply:
@XDelphiGrl I think this piece of code was not up to date, because I can't see it anymore. Sorry for the confusion!